libglusterfs/src/glusterfs/gf-dirent.h

```diff
@@ -15,6 +15,7 @@
 #include "glusterfs/inode.h"
 
 #define gf_dirent_size(name) (sizeof(gf_dirent_t) + strlen(name) + 1)
```

Comment:
This could be redefined as:
```suggestion
#define gf_dirent_size(name) gf_dirent_len(strlen(name))
```